src/main/java/com/boot/bankservice/service/impl/TransactionServiceImpl.java

```diff
@@ -22,8 +22,8 @@
     private final AccountService accountService;
 
     @Override
```

Comment:
it would be great to use `org.springframework.transaction.annotation.Transactional` here above the method

see more https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/transaction/annotation/Transactional.html

Reply:
Bohdan was talking about it, when doing two or more transaction in one method we should use this annotation